torch/nn/attention/flex_attention.py

```diff
@@ -173,13 +173,13 @@ def create_dense_one(kv_num_blocks, kv_indices):
     return out
 
 
-def _dense_to_ordered(dense_mask) -> Tuple:
+def _dense_to_ordered(dense_mask) -> Tuple[Tensor, Tensor]:
```

Comment:
might need to allow the `Tensor` to be optional to fix the lintrunner-noclang error below

otherwise looks good to me